### DISCUSSION TITLE
Tables on Markdown

### DISCUSSION THREAD

**Discussion by kt-brandi-larue** [community/community#108096] [New to GitHub]
I am new to GitHub and Markdown so this question might be pretty basic. I am trying to figure out to render tables on a .mdx file. 

All the examples and resources ive found online so far are not working. 

For example, when I try to write a table like this:

### Table
| Syntax      | Description |
| ----------- | ----------- |
| Header      | Title       |
| Paragraph   | Text        |

It get this on my port:

Table

| Syntax | Description | | ----------- | ----------- | | Header | Title | | Paragraph | Text |

Anyone know what I'm doing wrong here?

**Comment by gil-son** [comment]
In Markdown, tables are created using a combination of pipes (|) and hyphens (-). The example you've provided is correct, and it should render as a table when viewed in a Markdown viewer or when rendered on platforms like GitHub.

Here's your example formatted for a table:

```
| Syntax     | Description |
| ---------- | ----------- |
| Header     | Title |
| Paragraph  | Text |
```


| Syntax     | Description |
| ---------- | ----------- |
| Header     | Title |
| Paragraph  | Text |

If it's not rendering correctly in your specific Markdown environment, make sure:

1. There are no leading or trailing spaces in each row.
2.  The rows are separated by a blank line before and after the table.

**Comment by angelusrivera** [comment]
As Gil said, you're gonna want to start and end with a pipe and use the hyphens to create the horizontal aspects. The pipes are going to be your columns, while the hyphens are going to separate the headers from the rest of the table! Like so: 

```md
| Syntax    | Description |
|-----------|-------------|
| Header    | Title       |
| Paragraph | Text        |
```

Making a table like this:
| Syntax    | Description |
|-----------|-------------|
| Header    | Title       |
| Paragraph | Text        |

Takeaways:
- Make sure you're starting and ending with pipes
- The header HAS to be followed by those hyphens so don't forget that!
